src/orders/components/OrderDetailsRefundTable/refunds.ts

```diff
@@ -24,15 +25,26 @@ export interface DatagridRefund {
   } | null;
 }
 
+const findLatestEventWithCreatedBy = (
+  eventGroup: TransactionEventFragment[],
+): TransactionEventFragment | null => {
+  return eventGroup.find(event => !!event.createdBy) || null;
+};
+
 const SUPPORTED_REFUNDS = new Set([
   TransactionEventTypeEnum.REFUND_SUCCESS,
   TransactionEventTypeEnum.REFUND_FAILURE,
+  TransactionEventTypeEnum.REFUND_REQUEST,
 ]);
 
 export const manualRefundsExtractor = (
   order: OrderDetailsFragment | undefined,
   intl: IntlShape,
 ): DatagridRefund[] | undefined => {
+  if (!order) {
+    return;
+  }
+
```

Comment:
issue: this is hacky - if this function cannot proceed without `order` then it is not optional, that param is required, and this function should be called only when the order is present and never return undefined